src/test/java/core/basesyntax/model/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.model;
+
+import core.basesyntax.service.InvalidUserDataException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private RegistrationService registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
```

Comment:
You don't need to create new service for each test. Initialize together with declaration or in ```@BeforeAll``` method